### PR TITLE
Add policy for nvme-stas

### PR DESCRIPTION
**Pull request by 5umm3r15**
Add interface to allow domain to write the dbus pid sock_file.

Resolves: rhbz#2111414

### CODE REVIEW

File: policy/modules/contrib/nvme_stas.if [hunk 1]
```diff
@@ -0,0 +1,61 @@
+
```
Comment (zpytela) [inline policy/modules/contrib/nvme_stas.if]: Seems to be an unnecessary blank line.

File: policy/modules/contrib/nvme_stas.te [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+corecmd_exec_bin(nvme_stas_t)
+
+dev_read_sysfs(nvme_stas_t)
+domain_use_interactive_fds(nvme_stas_t)
+
+files_getattr_all_files(nvme_stas_t)
+files_read_etc_files(nvme_stas_t)
+
+nvme_stas_dbus_chat(nvme_stas_t)
```
Comment (zpytela) [inline policy/modules/contrib/nvme_stas.te]: This domain really needs dbus chat with itself?

**Review by zpytela — CHANGES_REQUESTED** [review]
Also an entry in modules.conf seems to be missing.

**Comment by zpytela** [comment]
Since the original bz has been closed migrated, I'd rather also have jira reference in the resolves field.

**Comment by 5umm3r15** [comment]
thanks for the review, fixed

File: policy/modules/contrib/nvme_stas.te (hunk 2 above)
Comment (zpytela) [inline policy/modules/contrib/nvme_stas.te]: Interface from the same module should not be used unless there is a good reason; this time it seems just a simple allow rule is needed.

**Comment by zpytela** [comment]
Merging, thank you.

**Pull request merged**